kolibri/plugins/learn/assets/src/views/thumbnails/ChannelThumbnail.vue

```diff
@@ -1,5 +1,4 @@
 <template>
-
```

Comment:
It seems like something is removing a lot of whitespace from these files - might be good to double check that your IDE is not applying its own formatting standards, that might well be conflicting with ours!